shell/platform/windows/flutter_window_unittests.cc

```diff
@@ -308,9 +312,9 @@ TEST(FlutterWindowTest, AccessibilityNodeWithoutView) {
 TEST(FlutterWindowTest, AlertNode) {
   std::unique_ptr<MockFlutterWindow> win32window =
       std::make_unique<MockFlutterWindow>();
-  ON_CALL(*win32window, GetPlatformWindow()).WillByDefault(Return(nullptr));
-  ON_CALL(*win32window, GetAxFragmentRootDelegate())
-      .WillByDefault(Return(nullptr));
+  EXPECT_CALL(*win32window.get(), GetAxFragmentRootDelegate())
+      .WillOnce(Return(nullptr));
```

Comment:
I'm assuming this is only called once? Would it be bad to be called more than once? If not, is there a way to check that it was called _at least_ once? That way, we'll be forced to clean up the mock if we ever stop needing to call it, but it'll be less fragile if we ever want to call it more than once.

Reply:
Yup we can use `WillRepeatedly` instead to verify >=1 calls. I'll update these!